google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -287,6 +294,26 @@ static Options fromQueryOptions(QueryOption... options) {
     return readOptions;
   }
 
+  static Options fromUpdateOptions(UpdateOption... options) {
```

Comment:
Can we add test for this?

Reply:
I've added a couple of (placeholder) tests for it. As this implementation does not contain any actual `UpdateOption` yet, the test case cannot check very much.